rs/nns/governance/src/governance/tla/merge_neurons.rs

```diff
@@ -0,0 +1,32 @@
+use super::{account_to_tla, extract_common_constants, post_process_trace};
+use crate::governance::governance_minting_account;
+use tla_instrumentation::{Label, TlaConstantAssignment, ToTla, Update, VarAssignment};
+
+pub fn merge_neurons_desc() -> Update {
```

Comment:
This is basically a constant, right? Maybe, consider using lazy_static. As a function, this can really do anything; in particular, it need not return the same thing every time. lazy_static is much more constrained. Engineering = putting the right constraints so that things (clearly!) do exactly what we want them to.

Reply:
Switched to a `lazy_static` here and in the other `_desc` implementations (split neuron, claim neuron).